base/src/test/resources/data-files/experiments_gxa/magetab/E-MTAB-3028/E-MTAB-3028.idf.txt

```diff
@@ -1,26 +1,27 @@
+Person Last Name	Lu	Regulski	Kendall	Reinders	Llaca	Deschamps	Smith	Levy	McCombie	Tingey	Rafalski	Hicks	Ware	Martienssen
+Person First Name	Zhenyuan	Michael	Jude	Jon	Victor	Stephane	Andrew	Dan	W	Scott	Antoni	Jim	Doreen	Robert
+Person Mid Initials									R					
+Person Email	luj@cshl.edu													
+Person Affiliation	Cold Spring Harbor Laboratory													
+Person Phone	6362884160													
+Person Address	Cold Spring Harbor Laboratory, 1 Bungtown Rd, Cold Spring Harbor, USA													
+Person Roles	submitter													
+Protocol Name	P-GSE39232-3	P-GSE39232-2	P-GSE39232-1											
+Protocol Description	Base call and quality scores were generated using the standard Illumina Pipeline. BS-seq reads were trimmed for adaptor sequence and low quality base call, then aligned to ZmB73 4a.53 genome assembly using rmap V2.03 with parameters -Q -v -m 4 Then the C/T counts from aligned BS-seq reads for each cytosine in reference genome were saved in CT count file RNA-seq reads were aligned to ZmB73 4a.53 and calculated FPKM using Tophat v1.3.2/Cufflinks v1.1.0 smRNA-seq reads were aligned to ZmB73 4a.53 genome aeembly using rmap V2.03 with parameters -Q -v -m 2, then the mapped 21/22/24 nt smRNA-seq reads were counted in each 1M bin in whole genome Genome_build: ZmB73 4a.53 Please note that the bed files for B73 DNA rep 1,2,3 and M017 DNA rep 1,2,3,4 Samples are on the Series record. Supplementary_files_format_and_content: The CT count files have the first 6 columns the same as bed file, the 7th col is count of C, the 8th col is count of T, the 9th col is the 3 nt in reference genome, the 10th col is the context of cytosine Supplementary_files_format_and_content: The FPKM tracking file format is described in http://cufflinks.cbcb.umd.edu/manual.html Supplementary_files_format_and_content: The BED from smRNA-seq data contains the count of mapped smRNA-seq reads in each 1M bin in the whole genome	Total RNA was prepared by grinding tissue in TRIzol reagent (Invitrogen 15596-026) on dry ice and processed following the protocol provided by manufacturer. To remove DNA, an aliquot of total RNA was treated with RQ1 DNase (Promega M6101), followed by phenol/chloroform/ iso-amyl alcohol extraction, chloroform/iso-amyl alcohol extraction and ethanol precipitation. Total RNA (20 μg) was used for poly(A)+ selection using oligo(dT) magnetic beads (Invitrogen 610-02), eluted in water and used for RNA-seq library construction with ScriptSeqTM kit (Epicentre SS10906) or mRNA-seq Sample Prep kit (Illumina RS-100-0801) according to the manufacturer’s protocol. DNA concentrations were quantified on a Bioanalyzer (Agilent), diluted to 10 nM and loaded on flow cells to generate clusters. Libraries were sequenced on Illumina GAII or HiSeq2000 machines using the paired-end 50 cycle protocol.	Seeds were grown in the incubator at 25�C in darkness on wet paper towels set in glass Pyrex dish. After 5 days coleoptiles were excised and stored at -80�C.											
```

Comment:
I asked curators to fix this file. Parts of it are encoded using UTF-8 other parts are encoded with ISO-8859-1/15.

Reply:
I also asked curators to fix E-MTAB-2848 but I didn't get a response.